examples/config-example.yaml

```diff
@@ -2313,6 +2569,14 @@ vgpu-configs:
           devices: all
           vgpu-devices:
             H20-48C: 2
+        - device-filter: "0x232C10DE"
+          devices: all
+          vgpu-devices:
+            H20X-70C: 2
+        - device-filter: "0x232C10DE"
+          devices: all
+          vgpu-devices:
+            H20X-70C: 2
```

Comment:
Note: there is a minor bug in the tool that generates this yaml file -- it will add duplicate entries in the `default` config entry whenever a device id (e.g. `0x232C10DE`) has multiple devices associated with it, each having a different subsystem ID. Having a duplicate entry here is not a functional issue, but it would be good to eventually update our config-gen tool to clean up duplicate entries in the `default` config.